documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+The benefits are following:
+1. Arch Board can review the ApiView when swagger PR is created and no need to wait for SDK owners to create ApiView manually, which saves much time.
```

Comment:
```suggestion
1. API signature could be reviewed at the same time when we submit a Swagger PR. This could help to detect API issues at the early stage.
```